### DISCUSSION TITLE
Axios and Secruity

### DISCUSSION THREAD

**Discussion by JHerwig** [nuxt/nuxt#9579] [Questions (Nuxt 2)]
A webpage with userspace communicates via Axios with an API.  How can I secure this communication? (Especially against man-in-the-middle attacks.) 

Does anyone know a method, a library, or a link that can help me out?

**Comment by TheAlexLichter** [comment]
Could you elaborate on your issue please? 🤔

**Reply by JHerwig** [reply]
1. I am looking for an established "Session" handling package. 
2. How can I prevent `Session Hijacking`.
3. Moreover are there further important security features I should know?
